File: src/test/R/examples.R
FUNZ_HOME="dist"
source(file.path(FUNZ_HOME,"Funz.R"))
Funz.init(FUNZ_HOME)

Funz_Run.info("R","src/test/samples/branin.R")

Funz_Run("R","src/test/samples/branin.R")

Funz_Run("R","src/test/samples/branin.R",input.variables = list(x1=runif(10),x2=runif(10)),verbosity=5)

File: src/test/python/examples.py
FUNZ_HOME=os.path.realpath("dist")
exec(open(os.path.join(FUNZ_HOME,"Funz.py")).read())
Funz_init(FUNZ_HOME,verbosity=10)

Funz_Run_info("R","dist/samples/branin.R")

Funz_Run("R","dist/samples/branin.R")

Funz_Run("R","dist/samples/branin.R",input_variables={'x1':[.1,.2,.3,.4],'x2':[.1,.2,.3,.4]},verbosity=5)

